src/Resources/Resources/help/Test-AzManagementGroupDeploymentStack.md

```diff
@@ -0,0 +1,632 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.ResourceManager.dll-Help.xml
+Module Name: Az.Resources
+online version:
```

Comment:
please fill out the online version

Reply:
please take a look at other files under this folder, the pattern should be obvious